modules/apps/29-fee/types/msgs.go

```diff
@@ -44,6 +46,21 @@ func (msg MsgRegisterCounterpartyAddress) GetSigners() []sdk.AccAddress {
 	return []sdk.AccAddress{signer}
 }
 
+// Route implements sdk.Msg
+func (msg MsgRegisterCounterpartyAddress) Route() string {
+	return RouterKey
+}
+
+// Type implements sdk.Msg
+func (msg MsgRegisterCounterpartyAddress) Type() string {
+	return TypeMsgRegisterCounterpartyAddress
+}
+
+// GetSignBytes implements sdk.Msg.
+func (msg MsgRegisterCounterpartyAddress) GetSignBytes() []byte {
```

Comment:
can you add a test which just ensure calling this function doesn't [panic](https://github.com/cosmos/ibc-go/blob/e2ac503744d7b432fdfd0237d1bf3a35f842b220/modules/apps/transfer/types/msgs_test.go#L59). We don't need to add the expected/actual equality check, that just becomes a hassle to maintain